src/glue-tables/alb-logs-table.ts

```diff
@@ -11,17 +11,49 @@ import { InputFormat, OutputFormat, SerializationLibrary } from '../glue/lib/dat
  * Configuration for AlbLogsTable
  */
 export interface AlbLogsTableProps extends ResourceProps {
+  /**
+   * A bucket where logs will be stored
+   * 
+   * @see [AWS S3 iBucket](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_s3.IBucket.html)
+   */
   readonly bucket: IBucket;
+  /**
+   * Boolean indicating whether to create Athena default Athena queries for the ALB Logs
```

Comment:
Seems like this is a typo. I assume the first `Athena` should be omitted.

This appears to be copy/pasted in the other files as well.

Reply:
Removed first Athena and updated to indicate the correct logs in each file.